server/src/instant/db/model/attr.clj

```diff
@@ -128,7 +128,7 @@
 ;; -------
 ;; caching
 
-(defonce attr-cache (cache/lru-cache-factory {} :threshold 256))
+(def attr-cache (cache/lru-cache-factory {} :threshold 256))
```

Comment:
Turns out to be nicer to make this `def` instead of `defonce` because if you change how the attrs are represented, then you automatically clear the cache when you reload the namespace.

Reply:
oh, makes sense!